packages/grid/_modules_/grid/hooks/features/pagination/useGridPage.ts

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { GridApiRef } from '../../../models';
+import { useGridSelector, useGridState } from '../core';
+import { useLogger } from '../../utils';
+import { GRID_PAGE_CHANGE, GRID_PAGE_SIZE_CHANGE } from '../../../constants';
+import { GridComponentProps } from '../../../GridComponentProps';
+import { useGridApiEventHandler, useGridApiMethod } from '../../root';
+import { GridPageApi } from '../../../models/api/gridPageApi';
+import { GridPageSizeState, GridPageState } from './gridPaginationState';
+import { visibleGridRowCountSelector } from '../filter';
+
+const getPageCount = (rowCount: number, pageSize: GridPageSizeState): number => {
+  if (pageSize > 0 && rowCount > 0) {
+    return Math.ceil(rowCount / pageSize);
+  }
+
+  return 0;
+};
+
+const clampPage = (pageState: GridPageState): GridPageState => {
```

Comment:
Can we rename it to something more meaningful and more common?

Reply:
What do you have in mind ?
I took the name of the lodash function that does the same thing : https://lodash.com/docs/#clamp

Reply:
I would have put something like `getValidPage` 🤔 
I was not familiar with `clamp`

Reply:
It is maybe a wording only used by `lodash`
Anyway, if it not clear to someone in the team, we should probably rename it :+1: